test/spec/PropertiesPanelSpec.js

```diff
@@ -686,6 +686,20 @@ describe('properties-panel', function() {
       }
     ));
 
+
+    it('should ignore incompatible provider', inject(function(propertiesPanel) {
+
+      // given
+      var incompatibleProvider = {
+        getGroups: function() {
+          return [];
+        }
+      };
+
+      // then
+      propertiesPanel.registerProvider(incompatibleProvider);
```

Comment:
Should we not check here that 'propertiesPanel.providersChanged' was not fired, that undefined was returned and potentially even that the error was logged?

Reply:
We should use internal `getProviders` API to verify the new provider is not part of the bundle.

I'll adjust the PR accordingly.